### DISCUSSION TITLE
Why is /workspaces/.codespaces/shared/editors/jetbrains installed in my Codespace?

### DISCUSSION THREAD

**Discussion by jeremysf** [community/community#102028] [Codespaces]
### Select Topic Area

Bug

### Body

It's taking up ~11GB of my 32GB Codespace disk.

I'm constantly getting out of disk space errors in my Codespace.

Can you help me understand what part of my Codespace configuration installed jetbrains? I'm not using jetbrains IDEs in any way and didn't knowingly install anything that seems like it would be related.

Thank you!

**Comment by CalvinLeather** [comment]
I'm experiencing the same. I have a repo that is <200Mb but here is the df outputs when the instance starts up:
![image](https://github.com/community/community/assets/6288302/d361c805-97a8-40bb-b2fa-e8fbb0dd8da2)
Disk report usage on github.com/codespaces:
![image](https://github.com/community/community/assets/6288302/8f1ee12c-4ecc-49a6-bff3-b8fbce3fea2d)
And then a du inside the directory in question:
![image](https://github.com/community/community/assets/6288302/3f482ac6-80bd-44a0-8a1c-b0de93ed3155)

Also, I noticed that between this and docker build cache stale layers, I was having some commands e.g., a gcc build and a pip install, fail due to insufficient disk space, and outputs reorted the same disk space reported by df. So it does seem this impacts users inside their container, and these files do impact usable space to the user.

Also, deleting these files doesn't seem to clear disk usage (some process must have access to them? or the way this directory/disk is mounted prevents deletion) and they re-appear when restarting the codespace

**Comment by andreped** [comment]
@jeremysf Did you manage to find a solution to this issue? I am experiencing the same.

**Comment by YoungjuneKwon** [comment]
I have just same problem. There are same folders in all of my codespaces, and I am looking for the way to remove the folders.

Does anyone have an idea?
